cdap-unit-test/src/main/java/co/cask/cdap/test/RuntimeStats.java

```diff
@@ -182,6 +192,15 @@ public String toString() {
 
   private static long getTotalCounter(Map<String, String> context, String metricName) {
     MetricDataQuery query = getTotalCounterQuery(context, metricName);
+    return getSingleValueFromTotals(query);
+  }
+
+  private static long getLatestGauge(Map<String, String> context, String metricName) {
```

Comment:
actually, maybe the question is why do we need this `getLatestGauge` method with the same implementation as `getTotalCounter` above it?